ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```
@@ -504,6 +505,13 @@ object BigDiffy extends Command {
       case m => throw new IllegalArgumentException(s"output mode $m not supported")
     }
 
+    if (om == GCS && !output.startsWith("gs://")) {
```

is it in scope to fail fast similarly for BQ?